database/apikeys.go

```diff
@@ -65,7 +81,7 @@ func (db *DB) APIKeyCreate(ctx context.Context, user User) (*APIKeyRecord, error
 	}
 	ak := APIKeyRecord{
 		UserID:    user.ID,
-		Key:       APIKey(base64.URLEncoding.EncodeToString(fastrand.Bytes(PubKeySize))),
+		Key:       APIKey(base32.HexEncoding.WithPadding(base32.NoPadding).EncodeToString(fastrand.Bytes(PubKeySize))),
```

Comment:
I'd move this up and use `LoadBytes`. That way we get the benefit of the length check. Right now this is bypassing any validation on the length of the key and is blindly casting to an `APIKey`.

Reply:
Ah, good catch. Will do.